### DISCUSSION TITLE
PocketBase as Framework with custom function in SDK

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2061] [Q&A] [answered]
If pockerbase is used as framework and custom functions are created then how to make request to custom function and handle response in dart sdk?

**Comment by ganigeorgiev** [comment, marked as answer]
You can call [`pb.send(...)`](https://pub.dev/documentation/pocketbase/latest/pocketbase/PocketBase/send.html).
